biopandas/pdb/tests/test_read_pdb.py

```diff
@@ -5,24 +5,28 @@
 # Code Repository: https://github.com/rasbt/biopandas
 
 
-from biopandas.pdb import PandasPdb
 import os
+
 import numpy as np
 import pandas as pd
-from nose.tools import raises
+from biopandas.pdb import PandasPdb
 from biopandas.testutils import assert_raises
+from nose.tools import raises
+
 try:
-    from urllib.request import urlopen
     from urllib.error import HTTPError, URLError
+    from urllib.request import urlopen
 except ImportError:
-    from urllib2 import urlopen, HTTPError, URLError  # Python 2.7 compatib
+    from urllib2 import HTTPError, URLError, urlopen  # Python 2.7 compatib
```

Comment:
I think we can actually remove this one. Due to the use of f-strings and type annotation elsewhere, i don't think biopandas can still run on Python 2.7